ci-operator/jobs/openshift/osde2e/openshift-osde2e-master-periodics.yaml

```diff
@@ -1088,6 +1088,53 @@ periodics:
   labels:
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: osde2e-stage-aws-addon-codeready-operator
+  spec:
+    containers:
+    - args:
+      - set -o pipefail; export HOME=$(mktemp -d); export REPORT_DIR=${ARTIFACTS};
+        ./ci/prow_run_tests.sh /usr/local/osde2e-common,/usr/local/crw-osd-token addons;
+      command:
+      - /bin/bash
+      - -c
+      env:
+      - name: ADDON_IDS
+        value: codeready-workspaces
+      - name: ADDON_TEST_HARNESSES
+        value: quay.io/crw/osd-e2e:2.1-latest
```

Comment:
I think it would be nicer to have just "latest" tag... To prevent changing this line with every (minor) release.

Reply:
Yes, you're right, I'll change to latest directly.